app/scripts/router.js

```diff
@@ -89,6 +89,7 @@ function (
       'cannot_create_account(/)': showView(CannotCreateAccountView),
       'verify_email(/)': showView(CompleteSignUpView),
       'confirm(/)': showView(ConfirmView),
+      'settings?setting=avatar': showView(AvatarChangeView),
```

Comment:
@zaach - :( This does not work for me locally, whether I try `http://127.0.0.1/settings?setting=avatar` or `http://127.0.0.1/settings/?setting=avatar`. The other thing about this approach is that if it did match, the AvatarChangeView would be displayed under two distinct URLs (settings?setting=avatar and settings/avatar/change).